app/main.py

```diff
@@ -16,4 +17,24 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+variables = [
+    ("lucky_number", lucky_number),
+    ("pi", pi),
+    ("one_is_a_prime_number", one_is_a_prime_number),
+    ("name", name),
+    ("my_favourite_films", my_favourite_films),
+    ("profile_info", profile_info),
+    ("marks", marks),
+    ("collection_of_coins", collection_of_coins)
+]
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
+for var_name, value in variables:
+    if isinstance(value, (list, dict, set)):
+        sorted_variables["mutable"].append(value)
+    else:
+        sorted_variables["immutable"].append(value)
+
+print(sorted_variables)
```

Comment:
don't use loop, just wirie data in dict manually

```python
sorted_variables = {
    "mutable": [my_favourite_films, ...],
    "immutable": [lucky_number, pi, ...]
}
```

Reply:
Do I just need to sort the data from the task into two groups? I thought i need to write code that will do it.